index.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="description" content="HABASM | HABASM365, Capstone" />
+    <meta name="keywords" content="Web design and Development, Capstone,Front-End,Front-End Developer,HABASM" />
+    <meta name="author" content="Capstone" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+    <link
+      rel="stylesheet"
+      href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="stylesheet" href="css/main.css" />
+    <link rel="stylesheet" href="css/desktop.css" />
+    <title>Green energy suitability</title>
+  </head>
+  <body>
+    <nav class="smallMenu">
+      <ul class="darkMenu">
+        <li>
+          <a href="#"
+            ><i class="fa fa-facebook social" aria-hidden="true"></i
+          ></a>
+        </li>
+        <li>
+          <a href="#"
+            ><i class="fa fa-twitter social" aria-hidden="true"></i
+          ></a>
+        </li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Profile</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </nav>
+    <nav class="bigMenu">
+      <ul>
+        <li>
+          <a class="logo" href="#"
+            ><img src="img/greenenergy.png" alt="Logo image"
+          /></a>
+        </li>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="#programs">Program</a></li>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#partners">Sponsor</a></li>
+        <li><a href="about.html">News</a></li>
+        <li><a class="webinar" href="#">Eneregy</a></li>
+      </ul>
+    </nav>
+    <section class="containerOne">
+      <button class="hamburger" type="button">
+        <i class="menuIcon material-icons">menu</i>
+        <i class="closeIcon material-icons">close</i>
+      </button>
+      <ul class="menu">
+        <li><a class="menuItem" href="index.html">Home</a></li>
+        <li><a class="menuItem" href="#programs">Program</a></li>
+        <li><a class="menuItem" href="about.html">About</a></li>
+        <li><a class="menuItem" href="#partners">Sponsor</a></li>
+        <li><a class="menuItem" href="about.html">News</a></li>
+      </ul>
+
```

Comment:
- [ ] Great job. you are right on track. please make the home page text a little centralize and aligned with the logo like in the behance design

yours
![Screenshot (612)_LI](https://user-images.githubusercontent.com/65298048/203498236-ec8ff43a-628e-4e2b-b971-2a6093e3595c.jpg)
behance
![Screenshot (613)_LI](https://user-images.githubusercontent.com/65298048/203498320-1ad1afcb-e095-47f9-bee9-2262ab8804f0.jpg)